tracing-flame/src/lib.rs

```diff
@@ -235,6 +255,36 @@ where
             out: self.out.clone(),
         }
     }
+
+    /// Configures whether or not periods of time where no spans are entered
+    /// should be included
+    ///
+    /// Defaults to true
+    ///
+    /// Setting this feature to false can help with situations where you have
+    /// large periods of time spent idling or otherwise doing uninteresting work
+    /// where no spans are active. These empty samples can pollute the final
```

Comment:
This is a really long sentence --- can we make it any more concise? Maybe
```suggestion
    /// Setting this feature to false can help with situations where no span is
    /// active for large periods of time. This can include time spent idling, or
    /// doing uninteresting work that isn't being measured.
```
or something?